### ISSUE TITLE
[CHANGE] when user joins airdrop use UID instead of name

### ISSUE THREAD

**Issue by realpaaao** [vultisig/vultisig-web#78]
To improve security of the user please use the 6 last letters of the UID of the registered vault instead of the vault name.
If user wants to they can change the vault name to be more distinct.

**Comment by Ehsan-saradar** [comment]
@realpaaao 
Right now use can change his vault alias from setting menu, do you still think we need to use uid ?!

**Comment by realpaaao** [comment]
@Ehsan-saradar yes! We need to have UID as standard when user uploads QR code and afterwards if they want to they can change it.
For security reasons.

**Comment by realpaaao** [comment]
@Ehsan-saradar As we discussed, please add option that when renamed they also rename on the leaderboard
